docs/design/keyboard-shortcuts.md

```diff
@@ -110,18 +143,17 @@ If there isn't one already, create a JSON file at the path that you use in devel
     }
     ```
 
-## Create a mapping of functions to actions
+## Create a mapping of actions to their functions
 
-The last major step is to map your custom functions onto the action names.
 
 1. Be sure that the HTML file that the `<FunctionFile>` element in the manifest points to has a `<script>` tag that loads a custom JavaScript file.
-1. In the JavaScript file, use calls of the [Office.actions.associate](/javascript/api/office/office.actions#associate) API to map a function to each action name that you used in the JSON file. To begin add the following to the file. Note the following about this code:
+1. In the JavaScript file, use calls of the [Office.actions.associate](/javascript/api/office/office.actions#associate) API to map a function to each action that you used in the JSON file. To begin add the following to the file. Note the following about this code:
 
-    - The first parameter is one of the action names from the JSON file.
-    - The second parameter is the function that runs when a user presses the key combination that is mapped to the action name in the JSON file.
+    - The first parameter is one of the actions from the JSON file.
+    - The second parameter is the function that runs when a user presses the key combination that is mapped to the action in the JSON file.
 
     ```javascript
-    Office.actions.associate('-- acton name goes here--', function () {
+    Office.actions.associate('-- action ID goes here--', function () {
```

Comment:
Again, wouldn't you put this in Office.onReady() so that it doesn't run before Office is ready?

Reply:
This API relies on Office.js to be loaded but not for Office.onReady(). For e.g.
<script src=”https://appsforoffice.microsoft.com/1.0/office.js” ></script>
<script>
              Office.actions.associate(“foo”, function() { });
</script>

Unlike other Rich APIs, it doesn't require Office.onReady(). It is similar to CustomFunctions.Associate() in technical behavior.